### PR TITLE
proc: Fix parsing of 'BuildVersion' on Go tip

### LINKED ISSUE #301: runtime.buildVersion is no longer available (Go tip)
`runtime.buildVersion` became `runtime/internal/sys.BuildVersion`

See https://github.com/golang/go/commit/432cb66f16b2bb6a167725057168bbe4aefe5fb5

### PR DESCRIPTION
**Pull request by derekparker**
Fixes #301

### CODE REVIEW

**Comment by derekparker** [comment]
Ping @aarzilli for review.

File: proc/proc.go [hunk 1]
```diff
@@ -704,8 +704,11 @@ func (dbp *Process) execPtraceFunc(fn func()) {
...
 	vv, err := dbp.EvalPackageVariable("runtime.buildVersion")
 	if err != nil {
-		err = fmt.Errorf("Could not determine version number: %v\n", err)
-		return
+		vv, err = dbp.EvalPackageVariable("runtime/internal/sys.BuildVersion")
+		if err != nil {
+			err = fmt.Errorf("Could not determine version number: %v\n", err)
+			return
+		}
 	}
```
Comment (kostya-sh) [inline proc/proc.go]: Would it be better to evaluate `runtime/internal/sys.BuildVersion` first and fallback to `runtime.buildVersion`? Go 1.6 will be released fairly soon. A comment mentioning that the name changed in Go 1.6 might be useful as well.

**Comment by aarzilli** [comment]
Seconding @kostya-sh note, minor nitpick though.

**Comment by derekparker** [comment]
Agreed, pushed change.

**Comment by aarzilli** [comment]
Merged ad5097a8dc716e61992d5c7dd6213e58cb0a88b5

**Pull request closed without merging**